TheMET/TheMET/ViewControllers/CatalogViewController.swift

```diff
@@ -86,19 +86,29 @@ class CatalogViewController: UIViewController {
                 completion(nil)
                 return
             }
-            self?.loadCatalogCellDataList(departments: responce.departments) { catalogCellDataList in
+            var catalogCellDataList: [CatalogCellData] = []
```

Comment:
- `completion` of this method doesn't make any sense anymore. 
- the method now interacts with `self.contentStatus` on its own.  It should do it in on its own instead of executing completion
- I assume the method can be renamed to `reloadCatalog`. Original `reloadCatalog` can be deleted.

Reply:
+